Chirp/Pages/MorePage/MorePageViewController.swift

```diff
@@ -75,4 +75,13 @@ extension MorePageViewController: UITableViewDelegate, UITableViewDataSource {
         cell.cellImage.image = UIImage(named: name)
         return cell
     }
+    
+    func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+        let rows: [CellNames] = [.pushNotification, .contributors, .logout]
+        if rows[indexPath.row].rawValue == "Contributors" {
```

Comment:
You can compare using the enum itself instead if the string.
```suggestion
        if rows[indexPath.row]== .contributors {
```